### PR TITLE
doc: fix missing document link

### PR DESCRIPTION
**Pull request by cncal**
Document section `hardware-requirements` locates to `/README.md`.

### CODE REVIEW

File: docs/install/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The following is an overview of the different installation methods available.
 ## Prerequisites
 
 Kata Containers requires nested virtualization or bare metal. Check 
-[hardware requirements](/src/runtime/README.md#hardware-requirements) to see if your system is capable of running Kata 
+[hardware requirements](/README.md#hardware-requirements) to see if your system is capable of running Kata
```
Comment (stevenhorsman) [inline docs/install/README.md:8]: I think if it's within the same file you don't need the file name
```suggestion
[hardware requirements](#hardware-requirements) to see if your system is capable of running Kata 
```

**Review by stevenhorsman — COMMENTED** [review]
Hi @cncal - thanks for the fix. I think it needs a slight tweak and also to comply with out commit format rules you need a commit message body and your `Signed-off-by`. Let us know if you need any help with adding that!

**Comment by cncal** [comment]
@stevenhorsman sorry about that and I have updated the commit, PTAL.

File: docs/install/README.md (hunk 1 above)
Comment (cncal) [reply docs/install/README.md:8]: Actually, the file `docs/install/README.md` does not have the `hardware-requirements` section for now.

Comment (stevenhorsman) [reply docs/install/README.md:8]: Ah sorry, I misunderstood, if you are pointing to the top-level README.md then you'll want `./../..//README.md#hardware-requirements`

Comment (cncal) [reply docs/install/README.md]: Updated.

Comment (stevenhorsman) [reply docs/install/README.md]: Thanks - looks good now!

**Review by stevenhorsman — APPROVED** [review]
LGTM now. Thanks for the updates!

**Review by stevenhorsman — CHANGES_REQUESTED** [review]
Sorry, can you fixup the fix into the first commit please?

File: docs/install/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ The following is an overview of the different installation methods available.
 ## Prerequisites
 
 Kata Containers requires nested virtualization or bare metal. Check 
-[hardware requirements](/src/runtime/README.md#hardware-requirements) to see if your system is capable of running Kata 
+[hardware requirements](./../..//README.md#hardware-requirements) to see if your system is capable of running Kata
```
Comment (stevenhorsman) [inline docs/install/README.md:8]: There is an extra slash here, but it still work at the moment
```suggestion
[hardware requirements](./../../README.md#hardware-requirements) to see if your system is capable of running Kata 
```

**Review by stevenhorsman — APPROVED** [review]

File: docs/install/README.md (hunk 2 above)
Comment (cncal) [reply docs/install/README.md]: updated.

**Review by danmihai1 — APPROVED** [review]

**Pull request merged**